db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_04_07_104659) do
+ActiveRecord::Schema[6.1].define(version: 2022_04_07_104659) do
```

Comment:
Might be worth running `rake db:schema:dump` now to update this to the Rails 7 version, otherwise there will be lots of changes next time someone touches the schema for real.